charts/victoria-logs-single/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,16 @@
+{{- $sa := .Values.serviceAccount }}
```

Comment:
Why does VL pod need a service acount?
It does not use any kind of access to k8s APIs, so it doesn't seem like it is needed

Reply:
Wdyt about disabling it by default than?
Seems like it might only be useful in some rare cases, so keeping it disabled by default will make impact on users minimal but will still allow to use it when needed.

Reply:
done